packages/scxa-marker-gene-heatmap/src/MarkerGeneHeatmap.js

```diff
@@ -102,11 +102,14 @@ const MarkerGeneHeatmap = (props) => {
         value: plotLineAxisPosition,
         zIndex,
         label: heatmapType !== `multiexperimentcelltypes` && {
-          text: splitCellTypeLabel.join(`<br/>`),
+          // tilted and trim very long labels for long cell types
+          text: splitCellTypeLabel.length <= 3 ? splitCellTypeLabel.join(`<br/>`) : splitCellTypeLabel.slice(0, 3).join(`<br/>`) + `...`,
```

Comment:
Can you extract this code please to a `const` and add a well descriptive name to it?
That way you can remove the comment 1 line above.
Thanks